src/modules/toaster.ts

```diff
@@ -1,5 +1,7 @@
 import { Position, Toaster } from '@blueprintjs/core'
 
-export const AppToaster = Toaster.create({
+const AppToaster = Toaster.create({
```

Comment:
Use lower case for modules.
Also, there is no need to call it **appToaster**
It can be just a **toaster**